lexibank_chenhmongmien.py

```diff
@@ -1,15 +1,14 @@
-import csv
+import collections
+from pathlib import Path
```

Comment:
`Path` should now be imported from `pathlib`, and no longer from `clldutils`.

Reply:
@MacyL, please check this comment